code/script.js

```diff
@@ -1,51 +1,270 @@
+//We put in the parameter time and give it contidions. This is our input that controls which output we get.
+//We also put in the parameter timeGreetings, which is an array of greeting phrases. This becomes our output, depending on the time variable.
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
-// Set up your eventlisteners here
+  botReply("Aloha what´s your Name?");
+
+};
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+//We welcome the user (based on current time) with a delay of 1s
 setTimeout(greetUser, 1000);
+ 
+
+//Denna funktion (eventListener) gör att när användaren klickar på submit, så körs funktion igång. 
+//Vi sätter en variabel på värdet vi får ut av firstName (från form input id i HTML) för att lättare återanvända den.
+form.addEventListener('submit', (event) => {
+  event.preventDefault()   //Hindrar från att formen autosparar innan vi har hunnit köra vår kod.
+  
+  const firstName = document.getElementById('name-input').value;
+  showMessage(firstName, 'user')
+           
+
+
+// this funcyion delays the next question
+  setTimeout(() => firstQuestion(firstName), 1000) 
+})
+
+
+//-- This function uses the first input after greetings (firstName) and adds a message from the bot containing that input. 
+//At the same time, we want to prep the input form area for the next question. We REPLACE (not add) the content in the formWrapper with new HTML.
+//question0
+
+const firstQuestion= (firstName) => {
+
+ botReply (`Welcome to the world of pinapples ${firstName}. What do you need today? Fun Fact, History or  order the pineapple?`)
+
+
+setTimeout ( () =>
+botReply ('First of all did you know that : The top of a pineapple, after cleaning and drying, can be planted in soil and a new plant will grow. '), 500)
+
+
+
+
+
+wrapper.innerHTML =`
+<button id="fun-fact" > Fun Fact </button>)
+<button id="history" > History </button>
+<button id="order" > Send me a pineapple!! </button>`
+
+//why has this eventlistneer started to show up as a bubble?
+document.getElementById('fun-fact').addEventListener('click', () => userReplyOne('fun'))
+
+
+document.getElementById('history').addEventListener('click', () => userReplyOne("history"))
+document.getElementById('order').addEventListener('click', () => answerThree("order"))
+
+}
+//answer and first interaction
+
+const userReplyOne = (chooseOne) => {  
+if (chooseOne === 'fun'){
+
+
+  setTimeout ( 
```

Comment:
i'm not sure here, but could the effect of this be better if you set the "setTimeout" to different times, so start with 1000, then 1500 and so on?  Since it should be reading the code chronological I think the setTimeOut should be different.